addons/website/static/tests/tours/html_editor.js

```diff
@@ -215,3 +217,48 @@ wTourUtils.registerWebsitePreviewTour('test_html_editor_scss_2', {
         },
     ]
 );
+
+registry.category("web_tour.tours").add("website_code_editor_usable", {
+    // TODO: enable debug mode when failing tests have been fixed (props validation)
+    url: "/web#action=website.website_preview",
```

Comment:
Fixed :)

I did this because I initially needed to change the url to enable debug mode. Maybe there should be a flag in `registerWebsitePreviewTour` instead.

Reply:
> Maybe there should be a flag

Indeed, we can add it once needed :+1: 

Don't forget to adapt / answer my other comment: https://github.com/odoo/odoo/pull/169413#discussion_r1666999022 (and if you did not miss it, maybe try to handle all comments before push-forcing the branch again, otherwise you are adding extra load to the runbot for no good reason :+1:)

Reply:
Sorry about that, I realized there was another comment too late. I'll do my best to avoid that next time!